### DISCUSSION TITLE
Late participation

### DISCUSSION THREAD

**Discussion by sanxing-chen** [snap-stanford/ogb#220] [General]
Hi there,

Thanks for organizing such an interesting contest!

I'm sorry that it was late that I saw this competition. I'm wondering if there's still a chance to participate in some forms? For example, if the registration is strictly closed that we can't be on the leaderboard for the contest, can we still submit and obtain the results on the test set?

**Comment by weihua916** [comment]
Hi!

Thanks for asking this. Unfortunately, you cannot participate in the KDD Cup without registration. However, we are planning to have hidden-test-set evaluation even after the KDD cup so that the community can continue to use/explore the datasets.
